### PR TITLE
Added link to openshift docs

### PR DESCRIPTION
**Pull request by dmuntima**
### Type of change

- Documentation

Added a link to the openshift upstream docs in the Getting Started section as we have used OCP terminologies in the following sections.

### CODE REVIEW

File: documentation/book/getting-started.adoc [hunk 1]
```diff
@@ -44,6 +44,9 @@ ifdef::Kubernetes[]
 endif::Kubernetes[]
 {OpenShiftLongName} user must have the rights to manage role-based access control (RBAC).
 
+For more information about {OpenShiftName} and setting up {OpenShiftName} cluster, see link:https://docs.openshift.com/container-platform/3.9/welcome/index.html[OpenShift documentation].
+
```
Comment (ppatierno) [inline documentation/book/getting-started.adoc]: the link contains the 3.9 version which over time means having a link to a stale OpenShift documentation. There is no way to put a link without a specific version and OpenShift website loads the last one?

Comment (dmuntima) [reply documentation/book/getting-started.adoc]: No, I did not find any link to the OCP documentation page. Adding a link to the OpenShift docs home page (https://docs.openshift.com/index.html) may cause more confusion.

Comment (scholzj) [reply documentation/book/getting-started.adoc]: I wonder what link should we have there ... the oldest supported (3.9) or the latest. Both are not optimal. At the same time there is nothing what would not work with OpenShift Dedicated for example. But I guess this is a start at least.

**Comment by laidan6000** [comment]
Looks good to me.

**Pull request merged**